flume/neore/rpm/SOURCES/init.d/cygnus

```diff
@@ -25,6 +25,8 @@
 
 PARAM=$1
 
+CYGNUS_INSTANCE=${2}
```

Comment:
If `$2` is not provided the script works like previous version: starting, stopping or viewing the status of all the instances configured.

Is not necessary to check if `$2` comes or not because if is not stablished is an empty variable and in start function, for example:

``` 
...
for instance in $(ls ${CYGNUS_DIR}/conf/cygnus_instance_${cygnus_instance}*.conf)
do
...
done
```

If var `${cygnus_instance}` is empty starts all instances, if not start only the instance/s that match with the pattern `cygnus_instance_${cygnus_instance}*.conf`.